dataactvalidator/config/supervisord.conf

```diff
@@ -20,6 +21,8 @@ directory=/data-act/backend/dataactvalidator/
 startsecs=1
 startretries=10
 user=root
+environment=PYTHONPATH=%(ENV_PATH)s:/data-act/backend
```

Comment:
The other configs had it so I had not messed with out of fear that it may break something xD